src/screens/home/components/searcher/Search.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import './style.css'
+
+const Search = () => {
+
+    return (
+        <div className="search">
+            <input type="text" name="" id="" placeholder="Search"/>
+            <svg stroke="currentColor" fill="none" stroke-width="0" viewBox="0 0 24 24" height="1em" width="1em" xmlns="http://www.w3.org/2000/svg"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"></path></svg>
+        </div>
+    )
+}
+
+export default Search
```

Comment:
Finally we don't implement the search box, you must remove this entirely component